README.md

```diff
@@ -526,6 +526,107 @@ E, [2022-10-09T11:28:29.976548 #15] ERROR -- : (try 5/10): Error for topic subsc
 
 Please see [Compression](#compression)
 
+### Consumer rebalance listener
+
+You may also want to set a consumer rebalance listener for whatever reason. A very simple listener that simply logs the partition assignment and rebalance events would look like this:
+
+```ruby
+class MyConsumerRebalanceListener
+  attr_reader :logger
+
+  def initialize(logger)
+    @logger = logger
+  end
+
+  def on_partitions_assigned(_consumer, list)
+    partitions = list.to_h.map { |_key, values| values.map(&:partition) }.flatten
+    logger.warn("Partitions assigned: #{partitions}")
+  end
+
+  def on_partitions_revoked(_consumer, list)
+    partitions = list.to_h.map { |_key, values| values.map(&:partition) }.flatten
+    logger.warn("Partitions revoked: #{partitions}")
+  end
+end
+
+Racecar.config.consumer_rebalance_listener do
+  MyConsumerRebalanceListener.new(logger)
+end
+```
+
+Note that the exact requirements for the consumer rebalance listener interface are set by rdkafka-ruby and may change from version to version.
+
+#### Liveness probe
+
+You can also take advantage of the consumer rebalance listener to implement a liveness probe for Kubernetes.
+
+The core idea is detecting lack of progress in specific partitions by 1. updating the timestamp on files representing activity in a partition and 2. checking the timestamp of said files in a Kubernetes probe.
+Since partitions can be rebalanced for a variety of reasons, some housekeeping is necessary and the consumer rebalance listener can be used for that purpose. That will avoid false positives due to partitions that have been re-assigned to a different Kafka consumer.
+
+``` ruby
+require 'fileutils'
+
+class MyConsumer < Racecar::Consumer
+  def perform(message)
```

Comment:
```suggestion
  def process(message)
```

Reply:
Thanks! I've amended the commit and force-pushed.